compiler/src/dotty/tools/dotc/config/Config.scala

```diff
@@ -140,7 +140,7 @@ object Config {
   final val LogPendingUnderlyingThreshold = 50
 
   /** How many recursive calls to isSubType are performed before logging starts. */
-  final val LogPendingSubTypesThreshold = 50
+  final val LogPendingSubTypesThreshold = 70
```

Comment:
I have some tests with size 24 tuples that triggers this limit, I'm going to revert this change and run these test without `-Yno-deep-subtypes`.